app/renderer/js/utils/request-util.ts

```diff
@@ -17,11 +17,14 @@ const logger = new Logger({
 
 let instance: null | RequestUtil = null;
 
-// TODO: TypeScript - Use ProxyRule for the proxy property
-// we can do this now since we use export = ProxyUtil syntax
+interface ProxyRule {
+	hostname?: string;
+	port?: number;
+}
```

Comment:
This is already defined in `proxy-util.ts`; the comment is suggesting we should import it, not redefine it.